src/components/views/elements/AppTile.js

```diff
@@ -86,8 +90,17 @@ export default React.createClass({
         });
     },
 
-    _onEditClick: function() {
-        console.log("Edit widget %s", this.props.id);
+    _onEditClick: function(e) {
+        console.log("Edit widget ID ", this.props.id);
+        if (e) {
```

Comment:
You'll always have an event if it's given to an onClick handler, so this probably isn't strictly necessary.

Edit: actually come to think of it, what default are you preventing anyway?